crates/shell/src/shells/murex.rs

```diff
@@ -0,0 +1,66 @@
+use super::Shell;
+use std::fmt;
+use std::path::{Path, PathBuf};
+
+#[derive(Clone, Copy, Debug)]
+pub struct Murex;
+
+impl Murex {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Self {
+        Self
+    }
+}
+
+impl Shell for Murex {
+    fn format_env_export(&self, key: &str, value: &str) -> String {
+        format!(r#"$ENV.{key}="{value}""#)
+    }
+
+    fn format_path_export(&self, paths: &[String]) -> String {
+        format!(r#"$ENV.PATH="{}:$ENV.PATH""#, paths.join(":"))
+    }
+
+    fn get_config_path(&self, home_dir: &Path) -> PathBuf {
+        home_dir.join(".murex_profile")
+    }
+
+    fn get_env_path(&self, home_dir: &Path) -> PathBuf {
+        home_dir.join(".murex_preload")
+    }
+
+    fn get_profile_paths(&self, home_dir: &Path) -> Vec<PathBuf> {
+        vec![
+            home_dir.join(".murex_preload"),
+            home_dir.join(".murex_modules/"),
```

Comment:
Let's remove this since this list needs to be files only.

Reply:
fixed..